packages/frontend/web-ui/src/common/http/helpers/buildSerializableResponse.ts

```diff
@@ -0,0 +1,10 @@
+import { Response } from '../../../../../../api/libraries/api-http-client/lib/cjs';
+
+export const buildSerializableResponse = (
+  response: Response,
+): { body: Response; statusCode: number } => {
```

Comment:
body is not a Response

Reply:
I guess it's an object but... How do you know the type of this object or its structure?